src/components/AsideHeader/components/CompositeBar/Item/Item.module.scss

```
@@ -281,6 +439,40 @@ $actionLeftRightMargin: 10px;
+            var(--gn-aside-header-item-height, #{variables.$item-height}) / 2 +
+                var(--gn-aside-header-menu-group-tree-connector-width, 16px) / 2 +
+                var(--gn-aside-header-menu-group-nested-icon-gap, var(--g-spacing-1))
+        );
+
+        /**
+         * Without nested icons, align item title with the parent group header title
+         * (icon column + title gap), not with the nested icon column.
+         */
+        &#{$class}_hide-icon {
+            padding-inline-start: calc(
+                var(--gn-aside-header-item-height, #{variables.$item-height}) +
```

Stylelint (17.14.0)

[error] Unexpected newline after "+" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)